packages/driver/cypress/e2e/commands/querying/querying.cy.js

```diff
@@ -919,6 +917,12 @@ describe('src/cy/commands/querying', () => {
   })
 
   context('#contains', () => {
+    it('should keep multiple contains() separate', () => {
```

Comment:
What does this test mean?

Reply:
I've changed the name of this test to `does not interfere with other aliased .contains()` and moved it further down in the file, but not sure how to succinctly capture what it's guarding against in the test title.

Added a longer explanation as a code comment; let me know if that's still not clear and I'll see what I can do.